airflow/www/.eslintrc

```diff
@@ -5,6 +5,32 @@
   "rules": {
```

Comment:
Should we also add [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint)?

Reply:
Yeah, I was playing around with it. Let me make sure it's in this PR.

Reply:
Added!